MagicalRecord/Categories/NSPersistentStore+MagicalRecord.h

```diff
@@ -21,7 +21,8 @@ OBJC_EXPORT NSString * __MR_nonnull const kMagicalRecordDefaultStoreFileName;
 + (void) MR_setDefaultPersistentStore:(MR_nullable NSPersistentStore *) store;
 
 + (MR_nullable NSURL *) MR_urlForStoreName:(MR_nonnull NSString *)storeFileName;
-+ (MR_nullable NSURL *) MR_cloudURLForUbiqutiousContainer:(MR_nonnull NSString *)bucketName;
++ (MR_nullable NSURL *) MR_cloudURLForUbiquitousContainer:(MR_nonnull NSString *)bucketName;
++ (MR_nullable NSURL *) MR_cloudURLForUbiqutiousContainer:(MR_nonnull NSString *)bucketName __deprecated;
```

Comment:
Would you mind using the macros that have been setup for this purpose? `MR_DEPRECATED_WILL_BE_REMOVED_IN_PLEASE_USE("MR_cloudURLForUbiquitousContainer:")` will tell the user _what_ they need to use instead of the deprecated method.